com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/MotionBlur.compute

```diff
@@ -271,25 +280,33 @@ void MotionBlurCS(uint3 dispatchThreadId : SV_DispatchThreadID)
             float  centralDepth = motionVecAndDepth.z;
 
             float4 accumulation = 0.0;
+            float alphaAccumulation = 0.0;
             for (uint i = 0; i < stepCount; ++i)
             {
 #if GUERTIN2014_DOUBLE_DIR
                 dir = ((i & 1u) == 0) ? centralDir : dir;
 #endif
                 float fw_Depth, bw_Depth;
                 float fw_MotionVecLen, bw_MotionVecLen;
-                float4 fw_Sample = ProcessSample(i, dir, invSampleCount, centerSampleUV, centralDepth, centralMotionVecLen, pixelToSampleScale, randomVal, -1, fw_Depth, fw_MotionVecLen);
-                float4 bw_Sample = ProcessSample(i, dir, invSampleCount, centerSampleUV, centralDepth, centralMotionVecLen, pixelToSampleScale, randomVal, 1, bw_Depth, bw_MotionVecLen);
+                float fw_Alpha, bw_Alpha;
+                float4 fw_Sample = ProcessSample(i, dir, invSampleCount, centerSampleUV, centralDepth, centralMotionVecLen, pixelToSampleScale, randomVal, -1, fw_Depth, fw_MotionVecLen, fw_Alpha);
+                float4 bw_Sample = ProcessSample(i, dir, invSampleCount, centerSampleUV, centralDepth, centralMotionVecLen, pixelToSampleScale, randomVal, 1, bw_Depth, bw_MotionVecLen, bw_Alpha);
 
                 // Mirror weights
 #if MIRROR_WEIGHTS
                 MirrorWeights(fw_Depth, bw_Depth, fw_MotionVecLen, bw_MotionVecLen, fw_Sample.w, bw_Sample.w);
 #endif
                 accumulation += float4(fw_Sample.w * fw_Sample.rgb, fw_Sample.w) + float4(bw_Sample.w * bw_Sample.rgb, bw_Sample.w);
+#if ENABLE_ALPHA
+                alphaAccumulation += fw_Sample.w * fw_Alpha + bw_Sample.w * bw_Alpha;
+#endif
             }
             accumulation *= invSampleCount;
 
-            colorAccumulation = accumulation.rgb + (1.0 - accumulation.w) * centralColor;
+            colorAccumulation.rgb = accumulation.rgb + (1.0 - accumulation.w) * centralColor.rgb;
```

Comment:
use CTYPE_SWIZZLE here 

colorAccumulation.CTYPE_SWIZZLE = accumulation.CTYPE_SWIZZLE + (1.0 - accumulation.w) * centralColor.CTYPE_SWIZZLE ;  instead of having #if ENABLE_ALPHA

Reply:
ok nevermind :)